### ISSUE TITLE
App crashes when selecting a time with an empty orderbook

### ISSUE THREAD

**Issue by Lercerss** [Lercerss/graphelier#63]
App crashes when selecting a time with an empty orderbook.

Steps:
- Select a valid time and see the orderbook displayed
- Select a time/date without a valid orderbook
- App crashes with following error:
```
TypeError: Cannot read property 'offsetTop' of null

MultiDirectionalScroll.handleVerticalScroll
src/components/MultiDirectionalScroll.js:57
> const topEdge = firstChild.offsetTop;

MultiDirectionalScroll.handleScroll
src/components/MultiDirectionalScroll.js:76
> this.handleVerticalScroll();
```

[//]: # (Make sure to add the appropriate feature label to your issue)

**Comment by chimano** [comment]
I'm having troubles recreating this bug on the current state of master. Does anyone have a reliable way of recreating it?

**Comment by alvyn279** [comment]
> I'm having troubles recreating this bug on the current state of master. Does anyone have a reliable way of recreating it?

+1

**Comment by alvyn279** [comment]
I recreate it by simply choosing our correct sample date, "21-06-2012".

- Choose "21-06-2012"
- Drag slider for new timestamp past 1PM, change the timestamp multiple times.. observe it crash.

**Comment by chimano** [comment]
I did a bit of investigation and found that the issue occurs when `MultiDirectionalScroll` doesn't have any  children. It can be fixed with a check to see if it has any children before doing anything in the `handleScroll` method

**Comment by alvyn279** [comment]
Solved in #84
